repo/blob/storage.go

```diff
@@ -17,6 +17,14 @@ var ErrSetTimeUnsupported = errors.Errorf("SetTime is not supported")
 // ErrInvalidRange is returned when the requested blob offset or length is invalid.
 var ErrInvalidRange = errors.Errorf("invalid blob offset or length")
 
+// TokenExpiredErrStr is the error string returned by the provider
+// when a token has expired.
+const TokenExpiredErrStr = "The provided token has expired"
+
+// ErrTokenExpired is returned when the token used for
+// authenticating with a storage provider has expired.
+var ErrTokenExpired = errors.Errorf(TokenExpiredErrStr)
```

Comment:
nit: can we call this `ErrInvalidCredentials` which is possible for username/password too when the password changes. Please adjust method names everywhere.

Reply:
(would be also great to adjust other providers' code if we can cleanly detect credential issues there)

Reply:
>  can we call this ErrInvalidCredentials which is possible for username/password too when the password changes. Please adjust method names everywhere.

Done. 

> (would be also great to adjust other providers' code if we can cleanly detect credential issues there)

Can I follow up on this in future PRs ?